frontend/packages/text-editor/src/TextList.test.tsx

```diff
@@ -59,44 +59,54 @@ const renderTextList = (props: Partial<TextListProps> = {}) => {
 };
 
 describe('TextList', () => {
-  test('updateEntryId should be called when id has been changed', async () => {
+  it('should call updateEntryId when the ID is changed in the edit mode', async () => {
     const user = userEvent.setup();
     const updateEntryId = jest.fn();
     const { rerender, initPros } = renderTextList({ updateEntryId });
     rerender(<TextList {...initPros} />);
+
     const toggleEditButton = screen.getAllByRole('button', { name: 'toggle-textkey-edit' });
     await act(() => user.click(toggleEditButton[0]));
-    const idInputs = screen.getAllByRole('textbox', {
-      name: 'tekst key edit',
-    });
-    await act(() => user.dblClick(idInputs[0]));
+    const idInput = screen.getByRole('textbox', { name: 'text key edit' });
+
+    await act(() => user.dblClick(idInput));
     await act(() => user.keyboard('a-updated{TAB}'));
     expect(updateEntryId).toHaveBeenCalledWith({ newId: 'a-updated', oldId: 'a' });
   });
 
-  test('that the user is warned when an id already exists', async () => {
+  it('should display warnings for existing, empty, or space-containing IDs', async () => {
     const user = userEvent.setup();
     const updateEntryId = jest.fn();
     const { rerender, initPros } = renderTextList({ updateEntryId });
     rerender(<TextList {...initPros} />);
+
     const toggleEditButton = screen.getAllByRole('button', { name: 'toggle-textkey-edit' });
     await act(() => user.click(toggleEditButton[0]));
-    const idInputs = screen.getAllByRole('textbox', {
-      name: 'tekst key edit',
-    });
-    const errorMsg = 'Denne IDen finnes allerede';
-    await act(() => user.dblClick(idInputs[0]));
+
+    const idInput = screen.getByRole('textbox', { name: 'text key edit' });
+    const errorMsg = [
```

Comment:
What do you think about naming this in plural, since it's an array with multiple error messages? 

```suggestion
    const errorMessages: string[] = [
```